HumanEvalLean/HumanEval51.lean

```diff
@@ -1,5 +1,50 @@
-def remove_vowels : Unit :=
-  ()
+import Std.Data.Iterators.Combinators.FilterMap
+
+def IsSubseq (s₁ : String) (s₂ : String) : Prop :=
+  List.Sublist s₁.toList s₂.toList
+
+def vowels : List Char := ['a', 'e', 'i', 'o', 'u', 'A', 'E', 'I', 'O', 'U']
+
+def no_vowels (s : String) : Prop :=
```

Comment:
Oops, missed that one in my earlier comment:
```suggestion
def NoVowels (s : String) : Prop :=
```

Reply:
got it and fixed it in bc1d67ceb470aebec5431dd3f2a51f4db38ded8f

It would be great to have a linter / formatter that will enforce conventions as much as possible.